### DISCUSSION TITLE
Group Manager on TAB module

### DISCUSSION THREAD

**Discussion by afiqsuradi** [Zrips/CMI#5940] [Feature Requests]
Hi,

   I'd love to request a feature for a tab module where I can **customize the tab according to the player's rank/group** just like [TAB](https://github.com/NEZNAMY/TAB). 
   [Here's](https://cdn.discordapp.com/attachments/860019002877018112/860153163726454794/unknown.png) the possibility of what tab module can do if this feature was added.
   
 Thanks.

**Comment by mrfloris** [comment]
https://github.com/Zrips/CMI/discussions/931
